content/intro-to-storybook/angular/en/screen.md

```diff
@@ -5,15 +5,15 @@ description: 'Construct a screen out of components'
 commit: 'b5f0de2'
 ---
 
-We've concentrated on building UIs from the bottom up; starting small and adding complexity. Doing so has allowed us to develop each component in isolation, figure out its data needs, and play with it in Storybook. All without needing to stand up a server or build out screens!
+We've concentrated on building UIs from the bottom up, starting small and adding complexity. Doing so has allowed us to develop each component in isolation, figure out its data needs, and play with it in Storybook. All without needing to stand up a server or build out screens!
 
-In this chapter we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
+In this chapter, we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
 
 ## Nested container components
 
-As our app is very simple, the screen we’ll build is pretty trivial, simply wrapping the `TaskListComponent` (which supplies its own data via ngxs) in some layout and pulling a top-level `error` field out of our store (let's assume we'll set that field if we have some problem connecting to our server).
+As our app is straightforward, the screen we’ll build is pretty trivial, simply wrapping the `TaskListComponent` (which supplies its own data via ngxs) in some layout and pulling a top-level `error` field out of our store (let's assume we'll set that field if we have some problem connecting to our server).
```

Comment:
```suggestion
As our app is straightforward, the screen we’ll build is pretty trivial, simply wrapping the `TaskList` component (which supplies its own data via ngxs) in some layout and pulling a top-level `error` field out of our store (let's assume we'll set that field if we have some problem connecting to our server).
```

I'm going to stop correcting these. The change to remove "Component" from component names was made in a file above, but applied inconsistently. Fwiw, I agree with the change.

Reply:
TaskListComponent, TaskComponent, and so on are removed from it. Thanks for this